Sources/Extensions/UIKit/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIStackViewExtensions.swift
+//  SwifterSwift-iOS
+//
+//  Created by Benjamin Meyer on 2/18/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+    
+// MARK: - Initializers
+@available(iOS 9.0, *)
+public extension UIStackView {
+    
+    /// SwifterSwift: Initialize an UIStackView with an array of UIView and common parameters.
+    ///
+    ///     let stackView = UIStackView(views: [UIView(), UIView()], orientation: .vertical)
+    ///
+    /// - Parameters:
+    ///   - views: The UIViews to add to the stack.
+    ///   - axis: The axis along which the arranged views are laid out.(default: .horizontal)
+    ///   - spacing: The distance in points between the adjacent edges of the stack view’s arranged views.(default: 0.0)
+    ///   - alignment: The alignment of the arranged subviews perpendicular to the stack view’s axis. (default: .fill)
+    ///   - distribution: The distribution of the arranged views along the stack view’s axis.(default: .fill)
+    public convenience init(views: [UIView], axis: UILayoutConstraintAxis = .horizontal, spacing: CGFloat = 0.0, alignment: UIStackViewAlignment = .fill, distribution: UIStackViewDistribution = .fill) {
```

Comment:
Sorry forgot about these points in my previous comment
- we want to rename `views` to `arrangedSubviews` as well.
- not give `axis` a default value since UIStackView already has the initializer `init(arrangedSubviews: [UIView])`

Thank you, again 😅